### DISCUSSION TITLE
Parsing file error

### DISCUSSION THREAD

**Discussion by Sleeper-guy** [Monika-After-Story/MonikaModDev#9568] [QA - tech support + general] [answered]
So I am trying to open game, I got some parsing file error in game\zz.calendar.rpy, and said some indentation error in some line, after that i searched up how to fix and found out i had to give spaces, i started doing those stuff, and now I think the whole file is broken cause it does not end giving errors from that file, currently its shows this error [
![image](https://user-images.githubusercontent.com/64907620/191598485-e102b9e1-da89-40a3-87ab-5f4d931239b9.png)
]
Please help me

**Comment by Booplicate** [comment, marked as answer]
We don't provide support for the source version. Make a new install and [download MAS from the releases](http://monikaafterstory.com/releases.html).
